autopilot/autopilot.go

```diff
@@ -84,55 +89,44 @@ func (ap *Autopilot) SetConfig(c Config) error {
 	return ap.store.SetConfig(c)
 }
 
-// Actions returns the autopilot actions that have occurred since the given time.
-func (ap *Autopilot) Actions(since time.Time, max int) []Action {
-	panic("unimplemented")
-}
-
 func (ap *Autopilot) Run() error {
 	if err := ap.load(); err != nil {
 		return err
 	}
 
-	go ap.contractLoop()
-	go ap.hostScanLoop()
-	<-ap.stopChan
-	return nil // TODO
+	for {
+		select {
+		case <-ap.stopChan:
+			return nil
+		case <-ap.ticker.C:
```

Comment:
I originally wanted to `range` over this ticker's channel but calling `Stop` on a ticker doesn't close the channel so you'd get into a deadlock. Figured the `stopChan` was the simplest way, it can also be used by the various subsystems to break early from semi-long-running loops (like renews).